### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**
[DEMO LINK](https://maksym-nemera.github.io/react_keyboard/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
```
Comment (dinaminahi) [inline src/App.tsx:9]: let's create a separate function for event

File: src/App.tsx [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+      this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
+    });
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.componentDidMount);
```
Comment (dinaminahi) [inline src/App.tsx:15]: and use it here instead of lifecycle method

**Review by dinaminahi — CHANGES_REQUESTED** [review]
Good job 👍 
but let's fix few moments

File: src/App.tsx [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount(): void {
```
Comment (dinaminahi) [inline src/App.tsx:8]: No need of specifying return `void` type, typescript can handle this by itself

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**